config/api.yml

```diff
@@ -2070,6 +2070,28 @@
       :delete:
       - :name: delete
         :identifier: iso_datastore_delete
+  :iso_images:
+    :description: ISO Images
+    :options:
+    - :collection
+    :verbs: *gp
+    :klass: IsoImage
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: iso_image_view
```

Comment:
Currently `iso_image_view` does not exist as a RBAC feature. 

I could either add it as a child of `iso_datastore_view` or use `iso_datastore_view` here.  @skateman Which way is better?

Reply:
cc @gtanzillo @Fryguy

Reply:
Initially, I was thinking that we could just use `iso_datastore_view` but since this includes `edit`, it would be better to add the new features as children of `iso_datastore_view`